site/js/server.js

```diff
@@ -0,0 +1,53 @@
+var Server = {
+    Automaton: {
+        create: function( data, callback, error ) {
+            $.post( 'api/automaton/create', {
+                name: '',
+                data: data
+            }, function( result ) {
+                callback( result );
+            }, 'json' ).error( function( jqXHR, settings, thrownError ) {
```

Comment:
Does not seem to work.. This code was written prior to me and I am not sure exacly why it is syntaxed like that.